### PR TITLE
Updated and added some new docs

### PR DESCRIPTION
**Pull request by thaigillespie**
Please remind me of what I need to do for Auth Docs (#60) issue again. I
keep forgetting so sorry : (

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # HKN Server
 
+## Deploys
+
+- `Production`: http://api.hknucsd.com/
```
Comment (godwinpang) [inline README.md:5]: Is endpoint protection on yet?

Comment (godwinpang) [inline README.md:5]: Is endpoint protection on yet?

**Review by godwinpang — APPROVED** [review]
Lgtm

File: README.md (hunk 1 above)
Comment (thaigillespie) [reply README.md:5]: Endpoint protection?

Comment (thaigillespie) [reply README.md:5]: @godwinpang what endpoint protection are you talking about?

Comment (godwinpang) [reply README.md:5]: uh auth; just wondering if we'll wake up and see 2k events in our db

Comment (thaigillespie) [reply README.md:5]: ohh uhhh not yet
I wasn't sure which auth middleware to put for each endpoint : p

Comment (godwinpang) [reply README.md:5]: ah ok let's not dos ourselves - can you put officerauth on the endpoints that aren't GET?

Comment (thaigillespie) [reply README.md:5]: for sure, on this PR or another?

Comment (godwinpang) [reply README.md:5]: another

Comment (thaigillespie) [reply README.md:5]: Oh and what about AuthController and OldUserController? What should I do with those?

Comment (thaigillespie) [reply README.md:5]: And also what about event sign in and rsvp endpoints? Aren't those supposed to be public?

Comment (godwinpang) [reply README.md:5]: ya event signin and rsvp endpoints are fine - we can leave those open

Comment (godwinpang) [reply README.md:5]: actually, what happens when someone signs up for a nonexistent event?

Comment (godwinpang) [reply README.md:5]: we can comment out the app.use(AuthRouter) and app.use(OldUserRouter) for now

Comment (thaigillespie) [reply README.md:5]: For sure

**Comment by thaigillespie** [comment]
This should be good then?

**Pull request merged**